data/abilities/setup/34bc0116-13b6-4dd5-b681-9554c2a1fa95.yml

```diff
@@ -52,7 +52,7 @@
         command: |
           New-Item -ItemType Directory -Force -Path C:\Users\Public\sensitive_file_backups | Out-Null;
           $output = '';
-          Get-ChildItem -Path #{file.sensitive.path} -EA silentlycontinue | foreach-object {
+          Get-Item #{directory.sensitive.path} -EA silentlycontinue | Where-Object { -not ($_ -is [System.IO.DirectoryInfo]) } | foreach-object {
```

Comment:
Not sure if file.sensitive.path should be changed to directory.sensitive.path

Reply:
Yup! Changed, thanks for the catch